internal/core/window.rs

```diff
@@ -309,6 +309,7 @@ impl crate::properties::PropertyDirtyHandler for WindowRedrawTracker {
 /// This enum describes the different ways a popup can be rendered by the back-end.
 enum PopupWindowLocation {
     /// The popup is rendered in its own top-level window that is know to the windowing system.
+    #[allow(dead_code)]
     TopLevel(Rc<dyn WindowAdapter>),
```

Comment:
I'd prefer this solution TBH (along with changing the use and call-sites).

```suggestion
    TopLevel { _adapter: Rc<dyn WindowAdapter> },
```